ansible/group_vars/all

```diff
@@ -207,6 +207,9 @@ docker:
   image:
     prefix: "{{ docker_image_prefix | default('whisk') }}"
     tag: "{{ docker_image_tag | default('latest') }}"
+  runtimes:
+    prefix: "{{ runtime_image_prefix | default('openwhisk') }}"
```

Comment:
isn't this already defined somewhere else for the runtime manifest (i'll look when I have a chance).

Reply:
Runtime manifest uses docker_image_prefix -- I wanted to avoid the overloading.

Reply:
@rabbah -- this what you were thinking of https://github.com/apache/incubator-openwhisk/blob/f3274d2155036dbd025902353f7b77c076ad1f86/ansible/group_vars/all#L203-#L210

Reply:
Yep.  Now we're going to differentiate between what's built locally by `./gradlew distDocker` (whisk/invoker, whisk/controller) and what's available for download as a "builtin" action `openwhisk/...`.  Of course, we should probably put some structure (tagged semver releases?) around the action runtimes in DockerHub.